build/kmath.js

```diff
@@ -1,628 +1 @@
-(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
FWIW, this is now a minified file, which I think is what we want for a build file.